analyzers/src/SonarAnalyzer.CSharp/Rules/UnusedPrivateMember.cs

```diff
@@ -225,14 +230,14 @@ private static Diagnostic GetDiagnosticsForProperty(IPropertySymbol property, IR
                 && property.SetMethod is { }
                 && GetAccessorSyntax(property.SetMethod) is { } setter)
             {
-                return Diagnostic.Create(RuleS1144, setter.GetLocation(), SyntaxConstants.Private, "set accessor in property", property.Name);
+                return Diagnostic.Create(RuleS1144, setter.Keyword.GetLocation(), SyntaxConstants.Private, "set accessor in property", property.Name);
             }
             else if (access == AccessorAccess.Set
                      && property.GetMethod is { }
                      && GetAccessorSyntax(property.GetMethod) is { } getter
                      && getter.HasBodyOrExpressionBody())
             {
-                return Diagnostic.Create(RuleS1144, getter.GetLocation(), SyntaxConstants.Private, "get accessor in property", property.Name);
+                return Diagnostic.Create(RuleS1144, getter.Keyword.GetLocation(), SyntaxConstants.Private, "get accessor in property", property.Name);
```

Comment:
Does this work for getters defined like this? I think `GetAccessorSyntax` fails short fo that kind of property declaration.
```cs
int Prop => 1;
```

Reply:
That's a good point, there were no test cases for it. So I've added one. Fortunately it seems to be working fine.
Tried it in SonarLint as well:

![image](https://user-images.githubusercontent.com/121798625/216097408-c99c0076-ca17-4c12-ad21-178549d04b65.png)